docs/walkthrough-1.7.md

```diff
@@ -0,0 +1,489 @@
+# Service Catalog Demonstration Walkthrough for Kubernetes v1.7.0 and Above
```

Comment:
That would depend on the target Kubernetes cluster. We need to do a better job in directing users to the right doc in the README, but I'd like to do that in a follow-up